HelpSource/Classes/History.schelp

```diff
@@ -56,44 +133,98 @@ store in a file, in historical order as individual code snippets.
 method::loadStory
 read history into current, from a file in story format.
 
+method::rewrite
+Write a properly formatted code file from a history.
+argument::path
+The filename is the original name with "_rewritten." appended.
+argument::open
+If open is true (default: true), open a text window with the string.
+
+method::readFromDoc
+read in a history from a code file created with .rewrite
+
+
+subsection:: editing history
+method::clear
+remove all items from (current) history.
+
+method::enter
+add an entry to (current) history by hand.
+
+method::drop
+drop the newest n lines from history. if n is negative, drop the oldest n lines.
+
+method::keep
+keep only the newest n lines from history. if n is negative, keep the oldest n lines.
+
+method::removeAt
+remove a specific line
+
+method::removeLast
+remove last line from history
+
+subsection:: repeating history
+method::evaluateLineAt
+
 method::play
 play back current history from start to end line, per default verbose.
 
 method::stop
 stop current history playback.
 
-method::rewrite
-Write a properly formatted code file from a history.
 
-argument::path
-The filename is the original name with "_rewritten." appended.
+subsection:: date, time, string functions
+method::startTimeStamp
+method::formatTime
+method::unformatTime
+method::shorten
+method::prettyString
 
-argument::open
-If open is true (default: true), open a text window with the string.
+private:: checkPath
 
-subsection::More internal methods:
+method::new
+create a new instance.
```

Comment:
what does the `lines` arg to `new` do? that needs to be documented

Reply:
done